docs/DeveloperGuide.md

```diff
@@ -316,13 +321,22 @@ The summary module is activated after the information is loaded from the main Vu
 #### Filtering users and repositories
 The commits information is retrieved from the corresponding project folders for each repository. These information will be filtered and sorted before passed into the template to be displayed as ramp charts.
 
-#### Padding for dates
-For ramps between the date ranges, the slices will be selected and it will be pre and post padded with empty slices to align the ramp slice between the `sinceDate` and `untilDate`. The ramps will then be rendered with the slices in the right position.
-
-### Authorship View (v_authorship.js)
+### Authorship View ([v_authorship.js](../frontend/src/static/js/v_authorship.js))
 The authorship module retrieves the relevant information from the corresponding `authorship.json` file if it is not yet loaded. If it has been loaded, the data will be written into `window.REPOS` and be read from there instead.
 
 ![authorship architecture](images/report-architecture-authorship.png)
 
 #### Showing relevant information by authors
-The files will be filtered, picking only files the selected author has written in. The lines are then split into chunks of "touched" and "untouched" code to be displayed in the tab view which will be popped up on the right side of the screen.
+The files will be filtered, picking only files the selected author has written in. The lines are then split into chunks of "touched" and "untouched" code segments to be displayed in the tab view which will be popped up on the right side of the screen.
+
+### Zoom View ([v_zoom.js](../frontend/src/static/js/v_zoom.js))
+The `v_zoom` module is in charge of filtering and displaying the commits from selected sub-range of a ramp chart. 
+
+### Ramp View ([v_ramp.js](../frontend/src/static/js/v_ramp.js))
+The `v_ramp` module is responsible for receiving the relevant information from `v_summary` and generating ramp charts that contain ramp slices.
+
+#### Padding for dates
+For ramps between the date ranges, the slices will be selected and it will be pre and post padded with empty slices to align the ramp slice between the `sinceDate` and `untilDate`. The ramps will then be rendered with the slices in the right position.
+
+### Segment View ([v_segment.js](../frontend/src/static/js/v_segment.js))
+The `v-segment` module is used as a component in `v_authorship`. It separates the code in terms of "touched" and "untouched" segments and only loads each "untouched" segment when it is toggled. 
```

Comment:
### Suggestion:

... (The module) distinguishes code segments that are touched by an author and those that are not. (The module) also handles the loading of untouched code segments when the user clicks on the `➕` icon.